src/main/java/seedu/address/logic/commands/AddBuyerProfile.java

```diff
@@ -10,39 +10,40 @@
 import seedu.address.logic.Messages;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
+import seedu.address.model.person.Buyer;
 import seedu.address.model.person.Person;
 
 /**
  * Adds a person to the address book.
  */
-public class AddClientProfile extends Command {
+public class AddBuyerProfile extends Command {
```

Comment:
Could be possible to extend AddBuyerProfile from AddClientProfile, prevents code reuse with AddSellerProfile